keepassxc-browser/popups/popup_httpauth.html

```diff
@@ -29,7 +29,10 @@
         <p>
           Select the login information you would like to get logged in with:
         </p>
-        <ul id="login-list"></ul>
+        <div id="login-list" class="list-group"></div>
+        <p>
+          <button id="btn-dismiss" class="btn btn-sm btn-danger"><span class="glyphicon glyphicon-remove"></span> Dismiss and show the manual authentication dialog</button>
```

Comment:
What is a 'manual authentication dialog'?

Reply:
Maybe it should be "default" instead.